buildSrc/src/main/java/com/osacky/flank/gradle/YamlWriter.kt

```diff
@@ -107,4 +118,20 @@ internal class YamlWriter {
       }
     }
   }
+
+  private fun isFlankVersionAtLeast(flankVersion: String, major: Int): Boolean {
+    val matchResult = FLANK_VERSION_REGEX.matchEntire(flankVersion)
```

Comment:
This solution works, but I would prefer if we tried to use the more idiomatic `VersionNumber` api in gradle. Can you try to see if this same thing can be achieved with `VersionNumber.parse(flankVersion).getMajor() > major`?

Reply:
Sure! Fixed in https://github.com/runningcode/fladle/pull/75/commits/db765b58b92d13b81aca7fe0365bc23daf9b9293

Reply:
Beautiful